ios/IJKMediaPlayer/IJKMediaPlayer/ijkmedia/ijksdl/ios/IJKSDLAudioQueueController.m

```diff
@@ -29,7 +29,7 @@
 
 #import <AVFoundation/AVFoundation.h>
 
-#define kIJKAudioQueueNumberBuffers (3)
+#define kIJKAudioQueueNumberBuffers (2)
```

Comment:
I reviewed ffplay's source code, just saw
#define VIDEO_PICTURE_QUEUE_SIZE 3
#define SUBPICTURE_QUEUE_SIZE 16
#define SAMPLE_QUEUE_SIZE 9
could you please tell me where "3 buffers is assumed in ffplay" ?